lib/logstash/inputs/snmp/client.rb

```diff
@@ -104,20 +104,24 @@ def walk(oid, strip_root = 0)
 
     private
 
+    NULL_STRING = "null".freeze
+
     def coerce(variable)
       variable_syntax = variable.getSyntax
       # puts("variable.getSyntaxString=#{variable.getSyntaxString}")
       case variable_syntax
-      when BER::OCTETSTRING
+      when BER::OCTETSTRING, BER::BITSTRING
```

Comment:
You have a require for `java_import "org.snmp4j.smi.OctetString"`. Do you need to add one for `BitString` as well?

Actually I see you don't have requires for the counters. Ok, I guess I don't have enough context here :-)

Reply:
Yeah I figured out that it's probably coming from the `asn1.BER` import. Never mind :-)

Reply:
the `OctetString` class is required for the `build_target` method - not needed for the coercion.